app/src/main/java/org/kiwix/kiwixmobile/utils/AlertDialogShower.kt

```diff
@@ -27,9 +28,19 @@ class AlertDialogShower @Inject constructor(
           clickListeners.getOrNull(0)
             ?.invoke()
         }
-        setNegativeButton(dialog.negativeMessage) { _, _ ->
-          clickListeners.getOrNull(1)
-            ?.invoke()
+        dialog.negativeMessage?.let {
+          setNegativeButton(it) { _, _ ->
+            clickListeners.getOrNull(1)
+              ?.invoke()
+          }
+        }
+        if (dialog === StartHotspotManually) {
```

Comment:
`dialog is StartHotspotManually`

Reply:
`dialog.neutralMessage?` this shouldn't be nullable

Reply:
Fixed